.gitpod.yml

```diff
@@ -1 +1 @@
-image: gitpod/workspace-dotnet:latest
+image: gitpod/workspace-dotnet:latest
```

Comment:
Same point as before. This should probably be `mcr.microsoft.com/dotnet/sdk:7.0`

Reply:
Ah this won't work due to the need to have node installed. Maybe we need to build on `mcr.microsoft.com/dotnet/sdk:7.0`?